src/main/java/net/snowflake/ingest/utils/HttpUtil.java

```diff
@@ -193,4 +195,40 @@ private static String getRequestUriFromContext(final HttpContext httpContext) {
     HttpRequest httpRequest = clientContext.getRequest();
     return httpRequest.getRequestLine().getUri();
   }
+
+  /**
+   * Helper method to decide whether to add any properties related to proxy server. These properties
+   * are passed on to snowflake JDBC while calling put API.
+   *
+   * @return proxy parameters that could be used by JDBC
+   */
+  public static Properties generateProxyPropertiesForJDBC() {
+    Properties proxyProperties = new Properties();
+    if ("true".equalsIgnoreCase(System.getProperty(USE_PROXY))) {
```

Comment:
Nit, is there some kind of `BooleanUtils` equivalent we can use here?

Reply:
Yes, updated